docs/product/relay/projects.mdx

```diff
@@ -108,3 +108,17 @@ An empty list rejects all origins. Use the default `["*"]` to allow all origins.
   }
 }
 ```
+
+`config.features`
+
+Enable product features that are not yet enabled by default.
```

Comment:
```suggestion
Some Sentry product features are disabled by default in Relay, either because they are still experimental or because they are not available on all plans. In static mode, these features have to be enabled manually if needed.
```